Instruments/notebooks/PileUpCompatibility.ipynb

```diff
@@ -0,0 +1,285 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Welcome to JupyROOT 6.12/06\n"
+     ]
+    }
+   ],
+   "source": [
+    "import ROOT\n",
+    "import re\n",
+    "import numpy as np\n",
+    "import operator\n",
+    "ROOT.enableJSVis()"
```

Comment:
why do you need `import operator`. Also can comment `ROOT.enableJSVis()`, since it is not used.